src/tools/w3dviewer/emitterlineproppage.h

```diff
@@ -0,0 +1,58 @@
+/**
+ * @file
+ *
+ * @author Jonathan Wilson
+ *
+ * @brief W3D View emitter line prop page
+ *
+ * @copyright Thyme is free software: you can redistribute it and/or
+ *            modify it under the terms of the GNU General Public License
+ *            as published by the Free Software Foundation, either version
+ *            2 of the License, or (at your option) any later version.
+ *            A full copy of the GNU General Public License can be found in
+ *            LICENSE
+ */
+#include "w3dafx.h"
+
+class EmitterInstanceList;
+
+class EmitterLinePropPageClass : public CPropertyPage
+{
+public:
+    EmitterLinePropPageClass();
+    virtual ~EmitterLinePropPageClass() override {}

```

Comment:
Can set `m_instanceList = nullptr` in destructor to document non owning pointer.